hack/e2e-test.sh

```diff
@@ -20,7 +20,7 @@ set -o pipefail
 
 SOURCE_DIR="$(cd "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)"
 ROOT_DIR="$SOURCE_DIR/.."
-export E2E_TEST_IMAGE=gcr.io/k8s-staging-perf-tests/sleep:v0.1.0
+export E2E_TEST_IMAGE=gcr.io/k8s-staging-perf-tests/sleep@sha256:8d91ddf9f145b66475efda1a1b52269be542292891b5de2a7fad944052bab6ea
```

Comment:
Could you please move it to `e2e-common.sh`? To do not duplicate it on hack/e2e-test.sh and hack/multikueue-e2e-test.sh.